cicecore/cicedynB/dynamics/ice_dyn_shared.F90

```diff
@@ -230,10 +232,10 @@ subroutine set_evp_parameters (dt)
       !dtei = c1/dte              ! 1/s
       dtei = real(ndte,kind=dbl_kind)/dt 
 
-      ! major/minor axis length ratio, squared
-      !ecc  = e_ratio**2
-      !ecci = c1/ecc               ! 1/ecc
-      ecci = c1/e_ratio**2               ! 1/ecc
+      ! variables for elliptical yield curve and plastic potential
+      epp2i = c1/e_plasticpot**2
```

Comment:
not sure I like that name.. but I don't have anything much better to propose :P

Reply:
At least it looks like the previous variable ecci...